### ISSUE TITLE
Section collapsing indicator color not working properly with action `Category`

### ISSUE THREAD

**Issue by davidlatwe** [pyblish/pyblish-qml#285]
## Problem
If I add `pyblish.api.Category` object in to `plugin.actions` list, the color of section collapsing indicator will not working as expected (Seems like plugin status got override to *default*).

## Test Code

Please try comment out `pyblish.api.Category("Fix"),` in each plugin's `actions` and see the change of section indicator `+`

```python
import pyblish.api


class DoSomeFix(pyblish.api.Action):
    label = "Make Some"
    on = "failed"

    def process(self, context, plugin):
        pass


class CollectSome(pyblish.api.ContextPlugin):

    order = pyblish.api.CollectorOrder - 0.2
    label = "Collect Some"
    actions = [
        #pyblish.api.Category("Fix"),
        DoSomeFix,
    ]

    def process(self, context):
        context.create_instance("something")


class CollectFailed(pyblish.api.ContextPlugin):

    order = pyblish.api.CollectorOrder - 0.2
    label = "Collect Failed"
    actions = [
        pyblish.api.Category("Fix"),
        DoSomeFix,
    ]

    def process(self, context):
        raise RuntimeError("Failed.")


class ValidateSome(pyblish.api.InstancePlugin):

    order = pyblish.api.ValidatorOrder - 0.2
    label = "Validate Some"
    actions = [
        pyblish.api.Category("Fix"),
        DoSomeFix,
    ]

    def process(self, instance):
        raise RuntimeError("Failed.")

```

**Comment by mottosso** [comment]
Thanks @davidlatwe, would you like to have a look at resolving this?

**Comment by davidlatwe** [comment]
Sure, will send a PR if I solved this.

**Comment by davidlatwe** [comment]
Solved with #286
Closing this one now:)
Thanks!
